finance_manager/src/main/java/com/exadel/finance/manager/user/controller/UserController.java

```diff
@@ -0,0 +1,40 @@
+package com.exadel.finance.manager.user.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.exadel.finance.manager.user.model.entity.User;
+import com.exadel.finance.manager.user.model.entity.UserModel;
+import com.exadel.finance.manager.user.service.api.UserService;
+
+
+@RestController
+@RequiredArgsConstructor
+public class UserController
+{
+
+    private UserService userService;
+
+    @GetMapping("/profile")
+    public ResponseEntity<User> readUser() {
```

Comment:
1) suggestion: method name getUserProfile()

2) Please never return pure Entity to Front part. Please use DTO (data transfer object)

3) How UserService will know which user to read? Should be some identifier
maybe 
public ResponseEntity<UserResponseDto> getUserProfile(@PathVariable Long userId) { //logic here}